modules/virt-configuring-masquerade-mode-dual-stack.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc
+
+[id="virt-configuring-masquerade-mode-dual-stack_{context}"]
+= Configuring masquerade mode with dual-stack (IPv4 and IPv6)
+
+You can configure virtual machines to use both IPv6 and IPv4 on the default Pod Network. The IPv6 network address must be statically set in the virtual machine.
+
+Once configured, incoming and outgoing traffic for the virtual machine is routed to both the IPv4 address and the unique IPv6 address of the virt-launcher Pod.
+The virt-launcher Pod then routes the IPv4 traffic to the DHCP address of the virtual machine, and the IPv6 traffic to the statically set IPv6 address of the virtual machine.
+
+This procedure can be applied to new or existing virtual machines in the cluster.
+
+.Prerequisites
+
+* The underlying network for the cluster must be connected with IPv6.
+
+* The OpenShift cluster must be configured to use IPv6.
```

Comment:
question: is there a specific configuration to use IPv6

Reply:
@RamLavi not sure at this stage. That work is being tracked on the OCP side and I'm following up with OCP docs.

Reply:
My understanding of OSDOCS-1640 is that ipv6 is configured by default going forward. Dropping the prereq.